database/test/db-insert.test.js

```diff
@@ -196,7 +196,8 @@ describe('Insert Update and Delete tests', function () {
       it('Should insert a single relationship', async function () {
         const insertResult = await postPersonaRoles.insertPersonaRole('dummy_tenant', 'https://dummytenant', 'dummy_subclaim', '', 'dummy_client', 'dummy_role:dummyA')
         assert.strictEqual(insertResult, 'Insert successful')
-        await postPersonaRoles.deletePersonaRole('dummy_tenant', 'https://dummytenant', 'dummy_subclaim', '', 'dummy_client', 'dummy_role:dummyA')
+        const deleteResult = await postPersonaRoles.deletePersonaRole('dummy_tenant', 'https://dummytenant', 'dummy_subclaim', '', 'dummy_client', 'dummy_role:dummyA')
+        assert.strictEqual(deleteResult, 'Delete successful')
```

Comment:
A quick thought, since this function does not throw an error ever, it would be better to return a `Promise<boolean>` instead of relying on string.

Reply:
@k1ch  yes I agree, though there are lots of functions like that which we should refactor to return a promise instead of relying on a string response